Sources/FoundationEssentials/Locale/Locale_Notifications.swift

```diff
@@ -0,0 +1,56 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(Synchronization)
+internal import Synchronization
+#endif
+
+/// Keeps a global generation count for updated Locale information, including locale, time zone, and calendar preferences.
```

Comment:
This new type stores the info about when we need to reset caches, due to system changes.